projects/recipe-page/phase-2-css-prompt.md

```diff
@@ -41,7 +41,7 @@ Phase 4 of the project involves incorporating jQuery. You'll add at least one in
 ## Setup Instructions
 Use the Terminal for navigating around the file system and creating new folders and files. Refer to the lesson on [The Command Line Interface](https://github.com/Techtonica/curriculum/blob/master/command-line/command-line-interface.md) if you need guidance on using the Terminal.
```

Comment:
Make the `The Command Line Interface` link relative.

Reply:
Am I supposed to edit this link? As I was asked to changed git related files only? @vegetabill